src/credentials/attestation_verifier/FileAttestationTrustStore.h

```diff
@@ -25,17 +25,29 @@
 namespace chip {
 namespace Credentials {
 
+enum class CertificateValidationMode
+{
+    // Validate that the certificate is a valid PAA certificate.
+    kPAA,
```

Comment:
I believe the function handles all three in the chain - can this be kAttestation or something similar?

Reply:
Nope,  I'm wrong, ignore that. Closing.

Reply:
> I believe the function handles all three in the chain

I don't think it does.  It explicitly calls:
```
VerifyAttestationCertificateFormat(certSpan, Crypto::AttestationCertType::kPAA)
```